integration/admin_session_fetch_blocks_test.go

```diff
@@ -41,16 +42,15 @@ func TestAdminSessionFetchBlocksFromPeers(t *testing.T) {
 	if testing.Short() {
 		t.SkipNow() // Just skip if we're doing a short run
 	}
+
 	// Test setup
-	testSetup, err := newTestSetup(newTestOptions())
+	tickInterval := time.Second
+	testSetup, err := newTestSetup(t, newTestOptions(t).SetTickInterval(tickInterval))
```

Comment:
Already have a property for it set to 3 seconds by default, will make the default 1 and see if that's sufficient for tests